src/System.Management.Automation/System.Management.Automation.csproj

```diff
@@ -16,6 +16,7 @@
     <PackageReference Include="System.IO.FileSystem.AccessControl" Version="4.4.0-preview3-25423-02" />
     <PackageReference Include="System.Security.AccessControl" Version="4.4.0-preview3-25423-02" />
     <PackageReference Include="System.Security.Cryptography.Pkcs" Version="4.4.0-preview3-25423-02" />
+    <PackageReference Include="System.Security.Permissions" Version="4.4.0-preview3-25423-02" />
```

Comment:
It turns out there is no corresponding implementation of `Zone.CreateFromUrl(string url)` ...
The implementation just returns null.